bentoml/saved_bundle/config.py

```diff
@@ -206,6 +208,12 @@ def get_bento_service_metadata_pb(self):
                     else:
                         api_metadata.mb_max_batch_size = DEFAULT_MAX_BATCH_SIZE
 
+                    # Supports customize route from 0.10.2
+                    if 'route' in api_config:
+                        api_metadata.route = api_config["route"]
+                    else:
```

Comment:
suggest adding a comment here `Use API name as the URL route when route config is missing, this is for backward compatibility for BentoML version <= 0.10.1`

Reply:
done